src/App.jsx

```diff
@@ -48,8 +50,23 @@ function App(props) {
     <Router>
       <div>
         {/* Protected Routes */}
-        <ProtectedRoute exact path="/" component={Home} />
+        <ProtectedRoute exact path="/" component={Home} view={"home"} />
         <ProtectedRoute path="/create-post" component={CreatePost} />
+        <ProtectedRoute
+          path="/side-bar"
+          component={Sidebar}
+          logout={props.logoutThunk}
+        />
+        <ProtectedRoute
+          path="/:username-posts"
```

Comment:
The proper way of linking REST endpoints would be `/username/posts`.
This is actually pretty important, and potential interview questions for designing REST APIs

Reply:
Ah ok, lets make sure to go over it then? some clarification on REST APIs would be nice I think

Reply:
Ill look into it in the meantime